func foobar() {
	return (x) ~ Error => {
		throw Error.new()
	}
}